MangaManager/src/Common/LoadedComicInfo/LoadedComicInfo.py

```diff
@@ -178,10 +184,22 @@ def _process(self, write_metadata=False, do_convert_to_webp=False, **_):
         try:
             with ArchiveFile(self.file_path, 'r') as zin:
                 assert initial_file_count == len(zin.namelist())
-            os.remove(self.file_path)
-            os.rename(tmpname, self.file_path)
-            logger.debug(f"[{'Processing':13s}] Successfully deleted old file and named tempfile as the old file",
-                               extra=self._logging_extra)
+            # If move_on_update setting is True, we should move the temp file to new location, then delete original file.
+            if Settings().get("Main", "move_on_update") == True:
```

Comment:
Can you help me understand this new code and why it would be used?

Reply:
Sorry! I thought the pull was matched to commit not pulling HEAD. Was adding file management functionality on my fork to aid in library management. Lemme close this out and make sure any file management code is removed.